marge/batch_job.py

```diff
@@ -297,10 +294,7 @@ def execute(self):
             except CannotMerge as err:
                 for merge_request in working_merge_requests:
                     merge_request.comment(
-                        'Batch MR !{batch_mr_iid} failed: {error} I will retry later...'.format(
-                            batch_mr_iid=batch_mr.iid,
-                            error=err.reason,
-                        ),
+                        'Batch MR !{batch_mr.iid} failed: {err.reason} I will retry later...',
```

Comment:
This is a bug, right? :thinking: Should be like this:

```suggestion
                        f'Batch MR !{batch_mr.iid} failed: {err.reason} I will retry later...',
```

Reply:
Thanks, good catch :+1: I wonder why CI didn't fail on this :thinking: I was of the impression that `pylint` would fail if there are `f-string`-looking strings that are not prefixed with `f`.. But maybe that was in a newer version.

Reply:
That would be really useful, in fact. Let's hope it comes in a future version yes.